### PR TITLE
text query html and solution

### PR DESCRIPTION
**Pull request by dakeljic-voysis**
Design for text query modal and calling previously written function to send the text query

### CODE REVIEW

File: test/client.html [hunk 1]
```diff
@@ -430,6 +461,9 @@ <h2 class="modal-card-title">Browser not supported</h2>
                   if (queryResult.conversationId) {
                       voysisClient.setQueryConversationId(queryResults.conversationId);
                   }
+                  sendButton.classList.remove('is-active');
```
Comment (jfarrelly-voysis) [inline test/client.html]: There's an error function just below this that you should add `sendButton.classList.remove('is-active');` to as well.  We want the send button to be deactivated whenever a query has finished, either successfully or unsuccessfully.

**Comment by jfarrelly-voysis** [comment]
You should change the `sendTextQuery` in `voysis.js` and remove the `skipCheckSessionToken` parameter.  When calling `sendEntityRequest` from `sendTextQuery` the value of `skipCheckSessionToken` should be `false`

File: test/client.html (hunk 1 above)
Comment (dakeljic-voysis) [reply test/client.html]: is that it? the one bellow?

Comment (jfarrelly-voysis) [reply test/client.html]: Yeah.  There's a lot of duplicated code now, but I think we can do a refactor in a separate PR.

**Pull request merged**